samples/app/MeshAssimp.cpp

```diff
@@ -213,6 +214,41 @@ Texture* MeshAssimp::createOneByOneTexture(uint32_t pixel) {
     return texturePtr;
 }
 
+void getMinMaxUV(const aiScene *scene, const aiNode* node, float2 &minUV, float2 &maxUV, int uvIndex) {
+    for (size_t i = 0; i < node->mNumMeshes; ++i) {
+        const aiMesh* mesh = scene->mMeshes[node->mMeshes[i]];
+        if (!mesh->HasNormals() || !mesh->HasTextureCoords(uvIndex)) {
```

Comment:
Why do you check `HasNormals` here?

Reply:
Good catch, this is a check that got copied from filamesh, not required here